src/components/Autocomplete/autocomplete.scss

```diff
@@ -0,0 +1,63 @@
+@import 'globals';
+
+$default-border-radius: map-get($form-fields-defaults, 'border-radius');
+
+.input {
+  @include map-declarations($form-fields-defaults, $form-fields-custom);
+  display: block;
+  width: 50%;
```

Comment:
I would try and use a wrapping component and always use `width: 100%` - get the wrapper define the width of the component